package.json

```diff
@@ -6,7 +6,10 @@
   "main": "index.js",
   "scripts": {
     "start": "node server.js",
-    "test": "node ace test"
+    "dev": "nodemon server.js",
```

Comment:
This change can come in a separate PR. All changes in this file can be reverted.